### PR TITLE
Add recipe for pdir2

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
Hmm it failed on AppVeyor because the build script couldn't find `curl`. Seems like a transitory problem with AppVeyor itself.

File: recipes/pdir2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - colorama
+    - enum34  # [py27]
```
Comment (jakirkham) [inline recipes/pdir2/meta.yaml]: Please drop anything that is not needed at build time. Probably just `python` and `setuptools`.

**Comment by jakirkham** [comment]
Yep, `curl` got moved recently on the VM. We are discussing how to best address this.

xref: https://github.com/appveyor/ci/issues/1426

File: recipes/pdir2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (nicoddemus) [inline recipes/pdir2/meta.yaml]: Done, my bad I didn't review that section more carefully after using `skeleton pypi`.

**Comment by nicoddemus** [comment]
Thanks for the link, I will follow that issue closely.

**Comment by jakirkham** [comment]
Missing AppVeyor. 😕 Toggling to try to get it back (especially now that it is fixed).

**Comment by mariusvniekerk** [comment]
All passed :shipit: ?

**Comment by jakirkham** [comment]
Yep, thanks for the nudge @mariusvniekerk. Also thanks for the contribution @nicoddemus.

**Pull request merged**